src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -83,7 +83,7 @@
 //! Target flags (test/bench/for_host/edition) | ✓           |                     |                        |
 //! -C incremental=… flag                      | ✓           |                     |                        |
 //! mtime of sources                           | ✓[^3]       |                     |                        |
-//! RUSTFLAGS/RUSTDOCFLAGS                     | ✓           |                     |                        |
```

Comment:
Seems like `unit_id` is generally the same as `c_extra_filename` at this moment. Do we anticipate in anything in the future of separating them? I think it worth some docs for future us and contributors.